src/ripple/nodestore/impl/DatabaseRotatingImp.cpp

```diff
@@ -102,19 +148,49 @@ DatabaseRotatingImp::sweep()
 std::shared_ptr<NodeObject>
 DatabaseRotatingImp::fetchFrom(uint256 const& hash, std::uint32_t seq)
 {
-    Backends b = getBackends();
-    auto nObj = fetchInternal(hash, b.writableBackend);
+    auto backends = [&] {
```

Comment:
Might be a good place for a structured binding (here and in the similar lambda in `for_each`). But fine as-is as well.

Reply:
Good idea, thanks!